.github/workflows/syntax-checks.yml

```diff
@@ -0,0 +1,72 @@
+name: Syntax Checks
+
+on: [push]
+
+jobs:
+  build-linux:
+    name: CI py${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.5', '3.6', '3.7', '3.8', '3.9']
+      max-parallel: 5
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Miniconda
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        activate-environment:
+            anaconda-test-env-py-${{ matrix.python-version }}
+        auto-update-conda: true
+        channel-priority: true
+        channels: defaults,conda-forge
+        # environment-file: environment.yml
+        mamba-version: "*"
+        python-version: ${{ matrix.python-version }}
+        use-only-tar-bz2: true  # required for caching
+    - shell: bash -l {0}
+      run: |
+        conda info
+        conda list
+        conda config --show-sources
+        conda config --show
+        conda env list
+        printenv | sort
+
+    # $CONDA is an environment variable pointing to the
+    # root of the miniconda directory
+
+    - name: Install dependencies
+      run: |
+        $CONDA/bin/conda env update --file environment.yml --name $ENV_NAME
+        # $CONDA/bin/conda install pytest --name $ENV_NAME
+        $CONDA/bin/conda list -r --name $ENV_NAME
+      env:
+        PY_VER: ${{ matrix.python-version }}
+        ENV_NAME: anaconda-test-env-py-${{ matrix.python-version }}
+
+    - name: Run tests
+      run: |
+        source $CONDA/bin/activate $ENV_NAME
```

Comment:
Can you use `conda activate`?

Reply:
Is `conda` known to the shell yet?  I'll check it.

Reply:
The answer is no, we need the line as shown.  The `conda` command is not yet known to the shell.  The `activate` line will add that to the shell and configure the other python toolset from the selected conda environment.

![Clipboard01](https://user-images.githubusercontent.com/2279984/99701380-00932a80-2a5a-11eb-91b3-027e42e1defd.png)